Makefile

```diff
@@ -15,6 +15,7 @@ LIBEXECDIR=$(PREFIX)/libexec
 BINDIR=$(PREFIX)/bin
 SBINDIR=$(PREFIX)/sbin
 VARLIBFQ=$(PREFIX)/var/lib/fq
+LUADIR=/share/lua/5.1
```

Comment:
I would leave out the leading `/` here, since you do things like `$(DESTDIR)/$(LUADIR)` below, and you'd end up with double slashes (which aren't fatal, but also are kinda ugly).

Reply:
fixed